test/unit/specs/store/__snapshots__/delegation.spec.js.snap

```diff
@@ -10,13 +10,13 @@ exports[`Module: Delegations fetches current undelegations 1`] = `
 Object {
   "cosmosvaladdr15ky9du8a2wlstz6fpx3p4mqpjyrm5ctplpn3au": Object {
     "balance": Object {
-      "amount": "356.00000000",
+      "amount": "-356.00000000",
```

Comment:
why is this minus?

Reply:
because of the unbonding that's defined below in `delegation.spec`:
`atoms: -356`

Reply:
I am pretty sure this is wrong. The returned values should be positive.
Example response from `GET /stake/delegations/{addr}`:
```
{"delegations":[{"delegator_addr":"cosmosaccaddr1cqz8e6ahj44fpnfmm4uvm04uxc4hytx3ugffpr","validator_addr":"cosmosaccaddr126ayk3hse5zvk9gxfmpsjr9565ef72pv9g20yx","shares":"28.0544622730","height":"482182"}],"unbonding_delegations":[{"delegator_addr":"cosmosaccaddr1cqz8e6ahj44fpnfmm4uvm04uxc4hytx3ugffpr","validator_addr":"cosmosaccaddr126ayk3hse5zvk9gxfmpsjr9565ef72pv9g20yx","creation_height":"0","min_time":"2018-09-26T15:53:10.620890836Z","initial_balance":{"denom":"steak","amount":"1"},"balance":{"denom":"steak","amount":"1"}}],"redelegations":null}
```

Reply:
fixed